src/main/java/com/lb/TripAgencySetup.java

```diff
@@ -0,0 +1,62 @@
+package com.lb;
```

Comment:
what is `lb` ?

Reply:
LB (lightbend)
I'll change it to something self evident. `tripagent`

Reply:
done